config/locales/ar/browse.yml

```diff
@@ -1,12 +1,19 @@
 ---
 ar:
   browse:
+    popular_links:
+      benefits:
+        - title:
+          url: /check-benefits-financial-support
```

Comment:
Is there any point adding the URL in all these places? I think it will just default to the `en` locale if the values are missing? So I'd be inclined to delete them to avoid confusion.

Reply:
It only makes sense, if we use translations, but as discussed earlier that's unlikely, so I'll remove them.